src/bin/inx-chronicle/cli/mod.rs

```diff
@@ -373,6 +417,25 @@ pub enum Subcommands {
         #[arg(long, value_name = "INPUT_SOURCE", default_value = "mongo-db")]
         input_source: InputSourceChoice,
     },
+    /// Fill interval analytics from Chronicle's database.
+    #[cfg(feature = "analytics")]
+    FillIntervalAnalytics {
+        /// The inclusive starting date (YYYY-MM-DD).
+        #[arg(short, long, value_parser = parse_date)]
+        start_date: Option<time::Date>,
+        /// The inclusive ending date (YYYY-MM-DD).
+        #[arg(short, long, value_parser = parse_date)]
+        end_date: Option<time::Date>,
+        /// The interval to use [day, week, month, year].
```

Comment:
The possible values are rendered automatically. How nice of `clap`.
```suggestion
        /// The interval to use.
```